charts/ocis/templates/_common/registry/registry.tpl

```diff
@@ -2,12 +2,16 @@
 oCIS service registry
 */}}
 {{- define "ocis.serviceRegistry" -}}
-{{- $valid := list "kubernetes" "etcd" "natsjs" -}}
+{{- $valid := list "kubernetes" "etcd" "nats-js-kv" -}}
```

Comment:
Should we remove `kubernetes` and `etcd` here? Basically depends on the product decision in https://github.com/owncloud/ocis/issues/7942

Reply:
Better to do this on next version. That way we can still use other registries if things go haywire.

Reply:
We're still on a major 0 release for the Helm Chart so I'd prefer to drop it now and add it back if other registries than `nats-js-kv` are production proven.

Reply:
dropped "kubernetes" but left "etcd". Latest comment I could find: "supported because it works"